File: acoustics_toolbox_r2020/Python/Bellhop/Tests/DW/Munk/MunkB_eigenray.py
#=======================================================================
# 
# Bellhop: Munk profile
# Mexilhoeira Grande, Sab Jul 29 14:19:39 WEST 2017
# Written by Tordar 
# 
#=======================================================================

from os import system
from numpy import *
from scipy.io import *
from pylab import *
sys.path.append ("/home/orodrig/FORdoc/at/Python/")
from plotray import *

print("Bellhop - Munk profile:") 
print("Eigenray trace run")
print("Geometric hat beams in Cartesian coordinates")

system("bellhop.exe MunkB_eigenray")

figure(1)
plotray('MunkB_eigenray.ray')
xlabel('Range (m)')
ylabel('Depth (m)')
title("Bellhop - Munk profile")

show()

print("done.")

File: acoustics_toolbox_r2020/Python/Bellhop/Tests/DW/Munk/MunkB_eigenray.m
clear all, close all

addpath /home/orodrig/FORdoc/at/Matlab/ReadWrite
addpath /home/orodrig/FORdoc/at/Matlab/Plot

disp('Bellhop - Munk profile:') 
disp('Eigenray trace run')
disp('Geometric hat beams in Cartesian coordinates')

system('bellhop.exe MunkB_eigenray');

plotray('MunkB_eigenray')

disp('done.')
